cypress/e2e/data-import/settings/attach-profiles-to-job-profile.cy.js

```diff
@@ -0,0 +1,142 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import NewJobProfile from '../../../support/fragments/data_import/job_profiles/newJobProfile';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import JobProfileView from '../../../support/fragments/data_import/job_profiles/jobProfileView';
+import NewActionProfile from '../../../support/fragments/data_import/action_profiles/newActionProfile';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+import { ACCEPTED_DATA_TYPE_NAMES } from '../../../support/constants';
+import Users from '../../../support/fragments/users/users';
+import ActionProfileView from '../../../support/fragments/data_import/action_profiles/actionProfileView';
+import MatchProfileView from '../../../support/fragments/data_import/match_profiles/matchProfileView';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const matchProfile1 = {
```

Comment:
It would be better to use an array of objects, like const collectionOfProfiles = [
      {
        mappingProfile: {
          name: `createInstanceMappingProf${getRandomPostfix()}`,
        },
        actionProfile: {
          name: `createInstanceActionProf${getRandomPostfix()}`,
        },
      },];
And use the element of array

Reply:
what array structure i can use when i need 3 action profiles 2 match profile 1 mapping profile and 1 job profile?

Reply:
please, join similar profiles (actions and match) to iterate actions for them

Reply:
fixed